packages/wallet/src/persistence/pouchDbStores/pouchDbWalletStores.ts

```diff
@@ -59,7 +59,7 @@ export const createPouchDbWalletStores = (
     destroyed: false,
     eraSummaries: new PouchDbEraSummariesStore(docsDbName, 'EraSummaries', logger),
     genesisParameters: new PouchDbGenesisParametersStore(docsDbName, 'genesisParameters', logger),
-    inFlightTransactions: new PouchDbInFlightTransactionsStore(docsDbName, 'transactionsInFlight', logger),
+    inFlightTransactions: new PouchDbInFlightTransactionsStore(docsDbName, 'transactionsInFlightV2', logger),
```

Comment:
```suggestion
    inFlightTransactions: new PouchDbInFlightTransactionsStore(docsDbName, 'transactionsInFlight.v2', logger),
```
Is this a legal name? Might be a little nicer to segment it this way

Reply:
How about using _? I think `.` is has too much potential to cause issues as it's used with different underlying stores. If you feel strongly about '.' I can test it with IndexedDB and LevelDB

Reply:
Happy with the underscore too. Just wanting to set a standard now since it's our first version bump

Reply:
43953b4be4f5fd744d3648fb71d6bf54eb384a9a